draft-opsarea-rfc5706bis.md

```diff
@@ -489,6 +494,9 @@ This document does not describe interoperability requirements. As such, it does
    changes, where they are required, and how they can be introduced in
    a manner that facilitates deployment.
 
+   Incentivizing good security operation practices when migrating to New Protocol should be encouraged. For example, patching is fundamental for security operations and can be incentivised if Protocol Designers consider supporting cheap and fast connection handoffs and reconnections. 
+
+   When Protocol Designers are considering how deployments should transition to New Protocol, impacts to current techniques employed by operators should be documented and mitigations included, where possible, so that consistent security operations and management can be achieved.
```

Comment:
```suggestion
   When Protocol Designers are considering how deployments should transition to the New Protocol or Protocol Extension, impacts to current techniques employed by operators should be documented and mitigations included, where possible, so that consistent security operations and management can be achieved.
```